backends/sentry.js

```diff
@@ -76,7 +76,7 @@ SentryBackend.prototype.createStream =
 module.exports = SentryBackend;
 
 function computeErrLoc(msg) {
-    var MAGIC_LINE_NUMBER_OFFSET = 13;
+    var MAGIC_LINE_NUMBER_OFFSET = 14;
```

Comment:
Since we keep tinkering can we write down a comment of what this magic number represents so that people of the future can at least guess if they should twiddle it.